tests/network/networkpolicy.go

```diff
@@ -40,50 +41,46 @@ var _ = SIGDescribe("[Serial][rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][l
 
 		tests.SkipIfUseFlannel(virtClient)
 		skipNetworkPolicyRunningOnKindInfra()
-
+		tests.BeforeTestCleanup()
```

Comment:
If the tests are properly isolated, you can remove the `Serial` tag now.

Reply:
Done